### PR TITLE
Add v0.22.0 release

### PR DESCRIPTION
**Pull request by lcarva**
# Release Notes

``` release-note
NONE
```

### CODE REVIEW

**Comment by lcarva** [comment]
@chitrangpatel , @wlynch ptal

File: releases.md [hunk 1]
```diff
@@ -100,6 +100,12 @@ deprecation window timers begin from when the release is published.
 
 ## Current Releases
 
+### v0.22 (LTS)
+- **Latest Release**: [v0.22.0][v0-22-0] (2024-08-19)
+- **Initial Release**: [v0.22.0][v0-22-0] (2024-08-19)
+
+[v0-22-0]: https://github.com/tektoncd/chains/releases/tag/v0.22.0
+
```
Comment (chitrangpatel) [inline releases.md:108]: Do you need to remove v0.19 (LTS) from support?

Comment (lcarva) [reply releases.md:108]: I don't think we can based on our [stability levels](https://github.com/tektoncd/chains/blob/821f7c446131a9cb303a8a9f63ced6ba17ca37fc/releases.md?plain=1#L57).

**Comment by lcarva** [comment]
@chitrangpatel review/lgtm? :pray:

**Review by robnester-rh — APPROVED** [review]
LGTM

**Pull request merged**